administrator/components/com_joomgallery/helpers/file.php

```diff
@@ -672,6 +796,7 @@ public static function resizeImage(&$debugoutput, $src_file, $dest_file, $usefor
           JoomFile::chmod($dir, '0755', true);
           if($return_var != 0)
           {
+            $debugoutput .= JText::sprintf('COM_JOOMGALLERY_UPLOAD_OUTPUT_IM_SERVERPROBLEM','exec('.$convert.');').'<br />';
```

Comment:
if the execution of the shell code fails, print out the shell code to debugoutput. In that case something with ImageMagick is missconfigured serversided.